### ISSUE TITLE
Safari/VoiceOver (macOS): VO focus is not moved when interacting with radio buttons and checkboxes with the mouse

### ISSUE THREAD

**Issue by querkmachine** [alphagov/reported-bugs#81]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=254591
**Date:** 2023-03-28
**Reported by:** @querkmachine 
**Related to:** https://github.com/alphagov/govuk-frontend/issues/3424

---

### Overview

In VoiceOver, it is usually possible to move VO's "focus" (that is, the content it will read out or control it will interact with) by clicking on it. For example, you can click on a word to have VO announce it, or click on a text input to have VO announce the input's label, hint text, and information about the input.

The current focus is indicated by a black outline that VoiceOver places around the element. 

This doesn't work as expected in two situations when using a mouse: checkboxes and grouped radio buttons.

### Expected result

In both cases, VO should move focus to the clicked input and announce information about the input. 

### Actual result

When clicking a checkbox that isn't already focused:

- The clicked checkbox will become checked.
- VoiceOver will not move focus to the clicked input.
- VoiceOver will not announce anything. 

When clicking a radio button that isn't already focused, in a group where a different radio is focused:

- The clicked radio button will become checked.
- VoiceOver will not move focus to the clicked input.
- VoiceOver will announce information about the **focused** radio button. It will not announce anything about the clicked radio button.
